### DISCUSSION TITLE
CareLink login error! Response code: 400

### DISCUSSION THREAD

**Discussion by mushmasha** [NightscoutFoundation/xDrip#2939] [General]
Hi everyone,
I am trying to use CarelinkFollower (both patient and follower accounts country is Ireland) as a datasource for xdrip 990df11-2023.05.26, but immediately getting an error CareLink login error! Response code: 400, Error nessage: NullPointerException:value==null,  this is happening for both patient and follower accounts, user names and passwords are 100% correct, MFA disabled. Any hints are appreciated!

Checked this one
https://github.com/NightscoutFoundation/xDrip/discussions/2821

But the behavior seems to be different.

**Comment by djgingersteve** [comment]
I'm having the same issue with a UK follower account. Patient name has been updated per the latest release, which initially worked, but now getting the 400 error

**Reply by mushmasha** [reply]
Fixed for me with xdrip nightly build of 8th of July 2023.
